V11.0.1 change log (#4848)
cypress-io/cypress-documentation: content/_changelogs/11.0.1.md

## 11.0.1

_Released 11/09/2022_

**Bugfixes:**

> - Fixed an issue that caused enabling experimentalSessionAndOrigin to throw a

Reviewer: ```suggestion
- Fixed an issue that caused enabling `experimentalSessionAndOrigin` to throw a
```

> - Fixed an issue that caused enabling `experimentalSessionAndOrigin` to throw a

Reviewer 2: Can we call out that these are fixing regressions in 11.0?

Author: Done

  webpack error. Fixes
  [#24611](https://github.com/cypress-io/cypress/issues/24611)

> - Fixed an issue where using the mochawesome reporter would cause Cypress to

Reviewer: ```suggestion
- Fixed an issue where using the `mochawesome` reporter would cause Cypress to
```

> - Fixed an issue where using the `mochawesome` reporter would cause Cypress to

Reviewer 3: ```suggestion
- Fixed an issue where using custom reporters would cause Cypress to
```

  throw a 'Cannot find module' error. Fixes
  [#24607](https://github.com/cypress-io/cypress/issues/24607)

> - Fixed testIsolation configuration validation to allow configuration updates

Reviewer: ```suggestion
- Fixed `testIsolation` configuration validation to allow configuration updates
```

  without restarting Cypress. Fixes
  [#24497](https://github.com/cypress-io/cypress/issues/24497).

> - Fixed an issue where cy.origin would not prevent users from setting up

Reviewer: lets add link to command
```suggestion
- Fixed an issue where `cy.origin()` would not prevent users from setting up
```

>   same-superdomain-origin cy.origin blocks. In these cases cy.origin is not

Reviewer: ```suggestion
  `same-superdomain-origin` `cy.origin()` blocks. In these cases `cy.origin()` is not
```

is same-superdomain-origin a Cypress term? If so, can we expand on what thtis is?
